platform/ios/src/MGLMapView.mm

```diff
@@ -3716,7 +3707,7 @@ - (void)setCamera:(MGLMapCamera *)camera withDuration:(NSTimeInterval)duration a
         };
     }
     
-    if ([self.camera isEqualToMapCamera:camera])
+    if ([self.camera isEqualToMapCamera:camera] && UIEdgeInsetsEqualToEdgeInsets(_contentInset, edgePadding))
```

Comment:
Here, `edgePadding` does include the content inset by default, but that’s a bug: #12818.

Reply:
https://github.com/mapbox/mapbox-gl-native/issues/12818#issuecomment-496836719
Let's continue discussion in #12818.